Hard limit for S3 artifact upload size (#2844)
buildkite/docs: pages/agent/v3/cli_artifact.md

> You can use this command in your build scripts to store artifacts. Artifacts are
> accessible using the web interface and can be downloaded by future build steps.

Reviewer: Re-writing this on a single line...

```suggestion
You can use this command in your build scripts to store artifacts. Artifacts are accessible using the web interface and can be downloaded by future build steps.
```

> Artifacts can be stored in the Buildkite-managed artifact store (a hard limit for artifact upload size is 5Gb for each upload), or your own storage location, depending on how you have configured your Buildkite Agent.

Reviewer: Thanks @karensawrey ,

Could you please confirm if the 5Gb limit relates to the size of a single artifact, or is it multiple artifacts totalling 5Gb in a single upload?

I'm assuming you mean the former and I'll suggest the following re-wording of this line to clarify that. However, I'll need confirmation about my question above before approving this PR, because if it's the latter, I'll need to change the wording again.

```suggestion
Artifacts can be stored in the Buildkite-managed artifact store, or your own storage location, depending on how you have configured your Buildkite Agent.

Be aware that the Buildkite-managed artifact store has an upload size limit of 5Gb for per artifact.
```

Reviewer: Actually, I confirmed the answer to my question above in the related ticket.

If this is all good with you, please accept my suggestions before merging, and I'll approve this PR in the meantime.
